### DISCUSSION TITLE
Fuzzy lookup to match column and bring results

### DISCUSSION THREAD

**Discussion by Dont-ask-for-it** [nocodb/nocodb#4348] [Q&A]
is this something possible with NocoDB? Starting from a database with column having similar content, and would like to match them using a score of similarity. Just like that:

https://www.youtube.com/watch?v=3v-qxcjZbyo

**Comment by dstala** [comment]
Not supported at the moment. You can raise a feature request [here](https://github.com/nocodb/nocodb/issues/new?assignees=o1lab&labels=%7B%22Type%22%3D%3E%22Feature%22%7D&template=--feature-request.yaml&title=%F0%9F%94%A6+Feature%3A+)

**Comment by Dont-ask-for-it** [comment]
Do you think it could be possible with a formula?

Le mer. 9 nov. 2022, 11 h 34 p.m., Raju Udava ***@***.***> a
écrit :
